app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt

```diff
@@ -86,9 +86,12 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class ProfileProgressFragmentTest {
 
-  @Inject lateinit var profileTestHelper: ProfileTestHelper
-  @Inject lateinit var storyProgressTestHelper: StoryProgressTestHelper
-  @Inject lateinit var context: Context
+  @Inject
+  lateinit var profileTestHelper: ProfileTestHelper
+  @Inject
```

Comment:
I think you need to do `reformat code` one more time so that an empty line appears between them else the ktlint will fail.

Reply:
I had not merged it with develop. I did it now please check.

Reply:
oh, ok